### DISCUSSION TITLE
Is Zed being built with the goal of taking over or being a real competitor to other IDEs like VS code and IntelliJ?

### DISCUSSION THREAD

**Discussion by 1366613** [zed-industries/zed#7946] [Feature Requests]
What's the bigger goal of this project?

**Comment by SomeoneToIgnore** [comment]
Zed provides some public channels, and the topmost one, `zed` (https://zed.dev/nightly/channel/zed-283) has the following in the channel notes:

```markdown
# Zed

Our mission is to fundamentally advance the experience of writing and collaborating on software.
```

While this might not be very elaborate, it matches my personal, general perception of the goals. I also think that, while it's pleasant to see how good we are in certain aspects compared to other editors already, the general goal should have nothing to do with "build a better editor than X" since it's very limiting, forcing us to frame the development into certain boundaries established by an editor X and catch-up with it — this is not a good goal at all.
